src/components/content/videos/VideoDetails.js

```
@@ -15,7 +15,12 @@ import useWindowWidth from '@/hooks/useWindowWidth';
 import dynamic from 'next/dynamic';
 import { Toast } from 'primereact/toast';
 import MoreOptionsMenu from '@/components/ui/MoreOptionsMenu';
-
+import React, { useEffect, useState, useRef } from 'react';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'React'. Consider to delete it or rename it.

'React' is defined here:


(lint/suspicious/noRedeclare)